app/brave_settings_strings.grdp

```diff
@@ -216,6 +216,30 @@
   <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_CONTENT_SECTION" desc="The label for section containing settings related to content">
     Content
   </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_CONTAINERS_SECTION" desc="The label for section containing settings related to containers">
+    Containers
+  </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_CONTAINERS_DESC" desc="The label for section containing settings related to containers">
+    Organize your browsing with containers. <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>Learn more<ph name="END_LINK">&lt;/a&gt;</ph>
+  </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_ADD_CONTAINER" desc="The label for dialog title for adding a container">
+    Add container
+  </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_EDIT_CONTAINER" desc="The label for dialog title for editing a container">
+    Edit container
+  </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_DELETE_CONTAINER" desc="The label for dialog title for deleting a container">
+    Delete container
+  </message>
+  <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_DELETE_CONTAINER_DESCRIPTION" desc="The description for dialog title for deleting a container">
+    Are you sure you want to delete container <ph name="CONTAINER_NAME">"$1"<ex>"Work"</ex></ph>? This will remove all the data associated with it.
```

Comment:
To improve the experience a little when users delete the default containers they don't want, we could first check whether or not a container has any data associated with it before we prompt the user.

- There is data in the container: PROMPT
- No data in container (e.g. in a new profile): delete without prompting